src/App.js

```diff
@@ -0,0 +1,337 @@
+    }
+
+  }
+  handleScissors() {
+    const computer = this.computerPlay()
+    const human = "scissors"
+    if (human === computer) {
+      this.setState({
+        computer,
+        human,
+        totalCount: this.state.totalCount + 1,
+        ties: this.state.ties + 1
+      })
+    }
+    if (human === "scissors" && computer === "rock") {
+      this.setState(
+        {
+          computer,
+          human,
+          computerCount: this.state.computerCount + 1,
+          totalCount: this.state.totalCount + 1
+        })
+    }
+    if (human === "scissors" && computer === "paper") {
+      this.setState({
+        computer,
+        human,
+        humanCount: this.state.humanCount + 1,
+        totalCount: this.state.totalCount + 1
+      })
+    }
+    if (human === "rock" && computer === "scissors") {
+      this.setState({
+        computer,
+        human, humanCount:
+          this.state.humanCount + 1,
+        totalCount: this.state.totalCount + 1
+      })
+    }
+    if (human === "rock" && computer === "paper") {
+      this.setState({
+        computer,
+        human,
+        computerCount: this.state.computerCount + 1,
+        totalCount: this.state.totalCount + 1
+      })
+    }
+    if (human === "paper" && computer === "rock") {
+      this.setState({
+        computer,
+        human, humanCount: this.state.humanCount + 1,
+        totalCount: this.state.totalCount + 1
+      })
+    }
+    if (human === "paper" && computer === "scissors") {
+      this.setState({
+        computer,
+        human,
+        computerCount: this.state.computerCount + 1,
+        totalCount: this.state.totalCount + 1
+      })
+    }
+  }
+
+  game() {
+    if (this.state.human === this.state.computer) {
+      return (<div>No Winner in this game</div>)
+    }
+    if (this.state.human === "scissors" && this.state.computer === "rock") {
+      return (<div>Computer Won this game</div>)
+    }
+    if (this.state.human === "scissors" && this.state.computer === "paper") {
+      return (<div>You Won this game</div>)
+    }
+    if (this.state.human === "rock" && this.state.computer === "scissors") {
+
+      return (<div>You Won this</div>)
+    }
+    if (this.state.human === "rock" && this.state.computer === "paper") {
+
+      return (<div>Computer Won this game</div>)
+    }
+    if (this.state.human === "paper" && this.state.computer === "rock") {
+
+      return (<div>You Won this game</div>)
+    }
+    if (this.state.human === "paper" && this.state.computer === "scissors") {
+
+      return (<div>Computer Won this game</div>)
+    }
+  }
+
+  handleReset() {
+    this.setState({
+      computer: "",
+      human: "",
+      status: "",
+      computerCount: 0,
+      humanCount: 0,
+      totalCount: 0,
+      ties: 0,
+    })
+  }
+
+  winner() {
+    const total = this.state.totalCount
+    const computer = this.state.computerCount
+    const player = this.state.humanCount
+    if (total % 5 === 0 && computer > player) {
```

Comment:
It would be easier to read if the first part of the conditional was simply `total === 5`